### PR TITLE
Added support for network mappings

### PR DESCRIPTION
**Pull request by ghost**
If the JSON input hash a `network_mappings` key, the wrapper loops over the mappings and adds a `--bridge source:destination` argument to the virt-v2v arguments. It keeps the default mapping to `ovirtmgmt`.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: files/virt-v2v-wrapper.py [hunk 1]
```diff
@@ -229,6 +229,10 @@ def wrapper(data, state_file, v2v_log):
             '-os', data['export_domain'],
             ])
 
+    if 'network_mappings' in data:
+        for mapping in data['network_mappings']:
```
Comment (nyoxi) [inline files/virt-v2v-wrapper.py:233]: please include also check for the type:  `isinstance(data['network_mappings'], list)`

log an error if the type does not match

File: files/virt-v2v-wrapper.py [hunk 2]
```diff
@@ -229,6 +229,10 @@ def wrapper(data, state_file, v2v_log):
...
+            v2v_args.extend(['--bridge', '%s:%s' % (mapping['source'], mapping['destination'])])
```
Comment (nyoxi) [inline files/virt-v2v-wrapper.py]: either check that both `source` and `destination` are in the dict, or use a safer way, e.g.: `mapping.get('source', '')`

**Review by nyoxi — CHANGES_REQUESTED** [review]

**Comment by nyoxi** [comment]
ci add to whitelist

**Review by nyoxi — APPROVED** [review]

**Comment by nyoxi** [comment]
ci test please

**Comment by ghost** [comment]
[Add network mappings support to virt-v2v wrapper](https://trello.com/c/gxB18dbu/117-add-network-mappings-support-to-virt-v2v-wrapper)

**Pull request merged**